src/Illuminate/Foundation/Http/FormRequest.php

```diff
@@ -65,6 +65,22 @@ class FormRequest extends Request implements ValidatesWhenResolved
      */
     protected $validator;
 
+    /**
+     * In case of first error, stop the validation process.
+     *
+     * @var bool
+     */
+    protected $failOnFirstError;
+
+    public function __construct(array $query = [], array $request = [], array $attributes = [], array $cookies = [], array $files = [], array $server = [], $content = null)
+    {
+        parent::__construct($query, $request, $attributes, $cookies, $files, $server, $content);
+
+        if (! property_exists($this->failOnFirstError)) {
```

Comment:
[property_exists docs](https://www.php.net/manual/en/function.property-exists.php)
But $this  already have `failOnFirstError` property. 

Why not just `protected $failOnFirstError = false;`?

Reply:
You are right. This questioning was unnecessary. Thanks.